src/components/SummaryList.scss

```diff
@@ -0,0 +1,42 @@
+.SummaryList-header {
+  color: $base-font-color;
```

Comment:
I wonder if this element could be simplified a bit. Maybe you could use flex on the parent and then the children wouldn't need classes or floats. Something like:
```
.SummaryList-heading {
  color: $base-font-color;
  font-size: $tu-large-fontSize;
  display: flex;  
  justify-content: space-between;
  margin-bottom: $su-xsmall;
}

<h3 class="SummaryList-heading">
  <span>Invoice</span>
  <span>Amount</span>
</h3>
```

PS:
+ We have a greedy attribute selector already for `[class*="-header"]` that applies a pseudo element to clear floats. This messes up the flex display because it considers this an equal child and tries to give it space. One work around is to add `flex-grow: 1` to a child. Another workaround is to just not use `header`... hence `heading` above. :eye-roll:
+  I added a font-size so that any heading element can be used and will display the same, unless we _want_ the size to be changeable.

Reply:
Used your example above and changed the class name.